docs/advanced/build_script.md

```diff
@@ -1,40 +1,40 @@
-# mbed SDK build script environment
+# mbed OS build script environment
 ## Introduction
-Mbed test framework allows users to test their mbed devices’ applications, build mbed SDK library, re-run tests, run mbed SDK regression, add new tests and get all this results automatically. Everything is done on your machine so you have a full control over compilation, and tests you run.
+mbed test framework allows users to test their mbed devices’ applications, build mbed OS library, rerun tests, run mbed OS regression, add new tests and receive all the results automatically. Everything occurs on your machine, so you have full control over compilation and the tests you run.
 
-It's is using Python 2.7 programming language to drive all tests so make sure Python 2.7 is installed on your system and included in your system PATH. To compile mbed SDK and tests you will need one or more supported compilers installed on your system.
+It uses Python 2.7 programming language to drive all tests, so make sure to install Python 2.7 and include it in your system PATH. To compile mbed OS and tests, you will need one or more supported compilers installed on your system.
 
-To follow this short introduction you should already:
-* Know what mbed SDK is in general.
-* Know how to install Python 2.7, ARM target cross compilers.
-* You have C/C++ programming experience and at least willingness to learn a bit about Python. 
+To follow this introduction, you should:
+* Know what mbed OS is in general.
+* Know how to install Python 2.7 and ARM target cross compilers.
+* Have C/C++ programming experience and be willing to learn a bit about Python. 
 
 ## Test automation
-Currently our simple test framework allows users to run tests on their machines (hosts) in a fully automated manner. All you need to do is to prepare two configuration files.
+Our test framework allows users to run tests on their machines (hosts) in a fully automated manner. All you need to do is prepare two configuration files.
```

Comment:
Is there a link to that bit of the instruction? Just to hint to people that we will explain this - it's not something that they should already know.